### ISSUE TITLE
Youtude.dll download link

### ISSUE THREAD

**Issue by akhtar122** [ytdl-org/youtube-dl#29709]
How can we get download link by running  Youtube-dll  exe in command prompt

C:\Users\User\Downloads>youtube-dll http:\\Anyvideiolink\pp---

**Comment by kikuyan** [comment]
There's no Youtude.dll or Youtube-dll.

**Comment by akhtar122** [comment]
Yes it is  - Youtube-dll

**Comment by rautamiekka** [comment]
No, the name doesn't have a dual small L, and no "`youtube-dll`" exists. I didn't research into "`youtube.dll`" more than what Google forced me to see in the results list but it seems it was a malicious thing, one unaffiliated with ytdl.

**Comment by akhtar122** [comment]
Yes you are right - youtube-dl 

C:\Users\User\Downloads>youtube-dl http:\Anyvideiolink\pp---
